projects/kit/components/text-area/text-area.template.html

```diff
@@ -45,10 +45,10 @@
                         [tuiBorderEnd]="border"
                     >
                         <span>{{fittedContent || controller.exampleText}}</span>
-                        <span class="pseudo-content__extra"
-                            >{{extraContent}}</span
-                        >
-                        &nbsp;
+                        <span
+                            class="pseudo-content__extra"
+                            [innerHTML]="extraContent"
```

Comment:
&nbsp; was there for a reason, it had some issues with line wrapping otherwise, not remember what it was

Reply:
fixed